scripts/o3de/o3de/project_properties.py

```diff
@@ -32,18 +30,18 @@ def _edit_gem_names(proj_json: dict,
                     new_gem_names: str or list = None,
                     delete_gem_names: str or list = None,
                     replace_gem_names: str or list = None,
-                    is_optional: bool = False):
+                    is_optional_gem: bool = False):
```

Comment:
styling question/nitpick: is it necessary to add the gem keyword here? I'm assuming that the function name _edit_gem_names provides the sufficient context.

Reply:
this was changed to be consistent with the other places `is_optional` was changed to `is_optional_gem`